tests/integration/test_store.py

```diff
@@ -1538,77 +1529,60 @@ def test_span_ingestion(
 
     spans = list(spans_consumer.get_spans())
     for span in spans:
-        del span["start_time"]
-        span["span"].pop("received", None)
+        span.pop("received", None)
 
     spans.sort(
-        key=lambda msg: msg["span"].get("description", "")
+        key=lambda msg: msg.get("description", "")
     )  # endpoint might overtake envelope
 
     assert spans == [
         {
-            "organization_id": 1,
```

Comment:
I think we still need the organization ID, right?

Reply:
We don't.

Reply:
No, we don't need `organization_id` anymore.

Reply:
Even if the field is described in the schema, the Snuba consumer doesn't need it to insert data: https://github.com/getsentry/snuba/blob/master/rust_snuba/src/processors/spans.rs#L28-L53.